src/FStarCompiler.fst.config.json

```diff
@@ -0,0 +1,3 @@
+{ "fstar_exe":"fstar.exe",
```

Comment:
A workspace config file, used by fstar-vscode-assistant. Useful for working on the compiler sources itself in vscode.